### ISSUE TITLE
Clerk.io module causing js-errors in frontend of PS 1.7.7.2

### ISSUE THREAD

**Issue by madsoliver** [clerkio/clerk-prestashop#6]
I get errors like:
```
Uncaught TypeError: $(...).staccordion is not a function
    at HTMLDocument.<anonymous> (theme.js:436)
    at c (core.js:39)
    at d (core.js:39)
```

..causing other functions in the PS frontend to stop.
Deactivating the module stops the errors.

Stack Owerflow answers is suggesting jQuery being loaded more than once can cause such errors.

**Comment by madsoliver** [comment]
Hi, Any update on this?

**Comment by madsoliver** [comment]
The errors started to occur after upgrade to PS 1.7.7.x
Can you do a test of the module is compatible with the latest PrestaShop?

**Comment by stubbedev** [comment]
This was resolved in version 6.5.0
